src/services/core/services-api/src/main/java/org/geoserver/geofence/services/dto/ShortRule.java

```diff
@@ -58,8 +60,9 @@ public ShortRule(Rule rule)
         setRequest(rule.getRequest());
         setWorkspace(rule.getWorkspace());
         setLayer(rule.getLayer());
-
+        setAddressRange(rule.getAddressRange().toString());
```

Comment:
This goes NPE if the rule address range is null.

Reply:
good point - I assumed Rule handled nulls but I should have checked